src/shared/components/Typography/Typography.tsx

```diff
@@ -1,10 +1,11 @@
 import { theme } from '../../theme';
-import { Body1, Body2, H1, H2, H3 } from './styles';
+import { Body1, Body2, Display, H1, H2, H3 } from './styles';
 import { TypographyProps } from './types';
 
 const typographyVariants = {
   Body1: Body1,
   Body2: Body2,
+  Display: Display,
   H1: H1,
   H2: H2,
   H3: H3,
```

Comment:
**Question:** O noticed that other components with variants are lowercase. Does it make sense to set a standard for variant names?

Reply:
That's fine to change this to lowercase too. But can I do it in another PR?

Reply:
for sure

Reply:
I added another one with this and other changes I was appalling to do: https://github.com/Wendlereis/minha-vez/pull/40